dbt/parser.py

```diff
@@ -75,10 +76,19 @@ def process_refs(flat_graph):
             elif len(ref) == 2:
                 target_model_package, target_model_name = ref
 
-            target_model = dbt.utils.find_model_by_name(
-                flat_graph,
-                target_model_name,
-                target_model_package)
+            # first, look for a model that matches the current
```

Comment:
@cmcarthur I'm kind of surprised this works! I think the ref resolution order we want is:
1. "top-level" project
2. The node's package
3. All packages

Does that seem right to you?